drivers/spi/Kconfig.stm32

```diff
@@ -21,6 +21,7 @@ config SPI_STM32_INTERRUPT
 config SPI_STM32_DMA
 	bool "STM32 MCU SPI DMA Support"
 	select DMA
+	select CACHE_MANAGEMENT
```

Comment:
This is required to use the functions used to keep cache coherency. See explanation in the dummy_rx_tx_buffer variable.